tests/unit/modules/network/a10/test_acos_config.py

```diff
@@ -71,30 +73,68 @@ def test_acos_config_lines(self):
         lines = ["ip dns primary 10.18.18.81"]
         set_module_args(dict(lines=lines))
         self.execute_module()
-        self.conn.edit_config.assert_called_with(
-            candidate=['ip dns primary 10.18.18.81'])
+        self.conn.get_diff = MagicMock(
```

Comment:
Why not just use the regular `Mock`? I don't see any magic methods being called upon.

Reply:
Yes, here Mock can be used. Replaced `MagicMock` with `Mock` now.
Resolved in PR#8